profiling_router.go

```diff
@@ -0,0 +1,146 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"strconv"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/go-utils/logger"
+)
+
+const PprofRoutePrefix = "/debug/pprof"
+
+type profiling struct {
+	Enable bool
+	Auth   authentication
```

Comment:
Is it needed that these fields are public?
```suggestion
	enable bool
	auth   authentication
```